baselines/dom.es6.generated.d.ts

```diff
@@ -0,0 +1,104 @@
+
+/////////////////////////////
+/// DOM ES6 APIs
+/////////////////////////////
+
+interface AudioTrackList {
+    [Symbol.iterator](): IterableIterator<AudioTrack>
+}
+
+interface CSSRuleList {
+    [Symbol.iterator](): IterableIterator<CSSRule>
+}
+
+interface CSSStyleDeclaration {
+    [Symbol.iterator](): IterableIterator<string>
+}
+
+interface ClientRectList {
+    [Symbol.iterator](): IterableIterator<ClientRect>
+}
+
+interface DOMStringList {
```

Comment:
does not seem itrable, https://www.w3.org/TR/DOM-Level-3-Core/core.html#DOMStringList